racket/collects/racket/match/define-forms.rkt

```diff
@@ -16,8 +16,9 @@
 (provide define-forms)
 
 (define-syntax-rule (define-forms parse-id
-                      match match* match-lambda match-lambda*
-		      match-lambda** match-let match-let*
+                      match match* match-lambda match-λ 
```

Comment:
Yes. Quite.
On Apr 5, 2016 22:16, "Benjamin Greenman" <notifications@github.com> wrote:

> In racket/collects/racket/match/define-forms.rkt
> <https://github.com/racket/racket/pull/1301#discussion_r58643203>:
>
> > @@ -16,8 +16,9 @@
> >  (provide define-forms)
> >
> >  (define-syntax-rule (define-forms parse-id
> > -                      match match* match-lambda match-lambda*
> > -		      match-lambda** match-let match-let*
> > +                      match match* match-lambda match-λ
>
> Is it possible to use (rename-out match-lambda match-λ) etc here? That
> would be simpler than using define-syntaxes
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/racket/racket/pull/1301/files/5162a876a38423b03e47c8a557e0cd7ca5a53ddb#r58643203>
>